Python/CMakeLists.txt

```diff
@@ -2,8 +2,6 @@
 
 option(INSTALL_PYTHON_INTERFACE "Build the Python interface" ON)
 
-find_package(Python3)
```

Comment:
Why is this removed?

Reply:
I just moved it to the start of the main CMake file so we can use python from anywhere in the code?

Reply:
But is it needed by anything else bar the Python stuff for Csound?

Reply:
Yeah, we need python for the command-line tests too

Reply:
Ok will do